RELEASE/scripts/autoscend/auto_community_service.ash

```diff
@@ -1030,9 +1027,8 @@ boolean LA_cs_communityService()
 			}
 			missing = min(2, missing);
 
-			if((have_effect($effect[Half-Blooded]) > 0) || (have_effect($effect[Half-Drained]) > 0) || (have_effect($effect[Bruised]) > 0) || (have_effect($effect[Relaxed Muscles]) > 0) || (have_effect($effect[Hypnotized]) > 0) || (have_effect($effect[Bad Haircut]) > 0))
-			{
-				doHottub();
+			foreach e in $effects[Half-Blooded, Half-Drained, Bruised, Relaxed Muscles, Hypnotized, Bad Haircut]{
+				uneffect(e);
```

Comment:
i suppose this pattern could lead to waste, maybe we should check for hot tub availability (though mafia should be smart enough to use hot tub if its available). In CS we wont have sgee but in other places we might and could burn a bunch unnecessarily if there wasnt a hot tub but we have a sgee on hand.